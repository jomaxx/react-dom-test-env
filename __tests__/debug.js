import React from "react";
import { mount, unmount, find, debug } from "react-dom-test-env";

afterEach(unmount);

it("pretty formats dom node", async () => {
  mount(
    <div>
      <button>test</button>
    </div>
  );

  expect(() => debug(find())).toThrowErrorMatchingSnapshot();
});
