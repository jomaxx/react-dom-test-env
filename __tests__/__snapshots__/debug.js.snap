// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pretty formats dom node 1`] = `
"<div>
  <button>
    test
  </button>
</div>"
`;
